client/src/Components/EventsPodium.tsx

```diff
@@ -22,245 +20,205 @@ export default function EventsPodium({ teams }: Prop) {
                                                 <img src={`${firstTeam.members[2].avatar}`} alt="" />
                                             </div>
                                         </div>
-                                        {/* <p className="podium__city">#3</p> */}
+                                    </div>
+                                    <div className="mt-10">
                                         <p className="podium__city">{firstTeam.members[2].name}</p>
                                         <p className="podium__city">{firstTeam.members[2].reg_no}</p>
                                     </div>
-                                )
-                                :
-                                (<></>)
+                                </div>
+                            ) : (<></>)
                         }
                     </div>
                     <p style={{ color: "white" }}>#1</p>
-                    <p style={{ color: "white" }}>{(firstTeam) ? firstTeam.name : ""}</p>
+                    <p style={{ color: "white" }}>{firstTeam ? firstTeam.name : ""}</p>
                 </div>
                 {
-                    (thirdTeam) ?
-                    (
-                            <div className='events-podium-container' style={{ display: "flex", flexDirection: "column" }}>
-                                <div className="container-events-podium container podium" style={{ transform: "translateY(50px)" }}>
-                                    {
-                                        (thirdTeam && thirdTeam.members.length >= 2) ?
-                                            (
-                                                <div className="podium__item" style={{ zIndex: "80" }}>
-                                                    <div style={{ height: "250px" }} className="podium__rank second">
-                                                        <div className="loader3">
-                                                            <span></span>
-                                                            <span></span>
-                                                            <span></span>
-                                                            <span></span>
-                                                        </div>
-                                                        <div className='image-cropper'>
-                                                            <img src={`${thirdTeam.members[1].avatar}`} alt="" />
-                                                        </div>
+                    thirdTeam ? (
+                        <div className='events-podium-container' style={{ display: "flex", flexDirection: "column" }}>
+                            <div className="container-events-podium container podium" style={{ transform: "translateY(50px)" }}>
+                                {
+                                    (thirdTeam.members.length >= 2) ? (
```

Comment:
your condition check should be uniform like every you are checking if the team exists or not and then you are checking for count but here you did not check that